frontend/app/components/Session_/StackEvents/UserEvent/userEvent.module.css

```diff
@@ -2,9 +2,6 @@
 .userEvent {
   border-radius: 3px;
   background-color: rgba(0, 118, 255, 0.05);
-  font-family: 'Menlo', 'monaco', 'consolas', monospace;
-  padding: 8px 10px;
-  margin: 3px 0;
```

Comment:
I restored the font. It looked really crazy because it was applied to the entire item, so I've switched it to the infoWrapper (so it only applies to what it applied to before I added a timestamp). 

The padding used for the items in console, exceptions and events (all the non table information) was different for each. I'm happy to use whatever, but I think they should all use the same padding, not something different everywhere.